tally/statgram.go

```diff
@@ -28,6 +29,7 @@ type Sample struct {
 	valueType   SampleType
 	sampleRate  float64
 	stringValue string
+	replace     bool
```

Comment:
As discussed in person, perhaps just having true/false variants of the gauge functions rather than taking a boolean would be a more readable approach.